src/storage/hummock_sdk/src/sstable_info.rs

```diff
@@ -33,6 +33,7 @@ pub struct SstableInfo {
     pub uncompressed_file_size: u64,
     pub range_tombstone_count: u64,
     pub bloom_filter_kind: PbBloomFilterType,
+    pub sst_size: u64,
```

Comment:
I am thinking whether we should have a function to expose `sst_size` for compaction instead of exposing this field directly. In this case, if we need to change the implementation of `sst_size`, we can only change the function implementation without changing other codes.

Reply:
It does work, but `sst_size` is a pub field, and currently all pub fields in the code are accessed directly.

I'm not sure we want to provide special access to `sst_size`, but I think the premise is that `sst_size` will no longer be a pub, otherwise you can't stop callers from not using it directly.